nvflare/app_common/pt/pt_fedopt.py

```diff
@@ -205,8 +212,15 @@ def shareable_to_learnable(self, shareable: Shareable, fl_ctx: FLContext) -> Lea
 
         model_diff = dxo.data
 
+        server_lr = None
+        hps = fl_ctx.get_prop(AppConstants.HYPERPARAMTER_COLLECTION)
+        if hps is not None:
+            server_lr = hps.get("slr")
```

Comment:
you can do 
`server_lr = fl_ctx.get_prop(AppConstants.HYPERPARAMTER_COLLECTION, {}).get("slr")`
to achieve the same thing

Reply:
@guopengf Please take a look.

Reply:
Yeah, Yuan-Ting is right. Removed the redundant code. https://github.com/NVIDIA/NVFlare/pull/1283/commits/9d6d968cf4593feea9d9fcb61e600256014952e9